ebpf_api doxygen warnings

The [latest doxygen pass](https://github.com/microsoft/ebpf-for-windows/runs/2897300734?check_suite_focus=true) reported the following warnings:
```
/home/runner/work/ebpf-for-windows/ebpf-for-windows/include/ebpf_api.h:91: warning: The following parameters of ebpf_api_create_map(ebpf_map_type_t type, uint32_t key_size, uint32_t value_size, uint32_t max_entries, uint32_t map_flags, _Out_ ebpf_handle_t *handle) are not documented:
  parameter 'map_flags'
  parameter 'handle'
/home/runner/work/ebpf-for-windows/ebpf-for-windows/include/ebpf_api.h:448: warning: argument 'program' of command @param is not found in the argument list of ebpf_map_get_fd(_In_ const struct _ebpf_map *map)
/home/runner/work/ebpf-for-windows/ebpf-for-windows/include/ebpf_api.h:455: warning: The following parameter of ebpf_map_get_fd(_In_ const struct _ebpf_map *map) is not documented:
  parameter 'map'
/home/runner/work/ebpf-for-windows/ebpf-for-windows/include/ebpf_api.h:357: warning: argument 'ebpf_object' of command @param is not found in the argument list of ebpf_program_load(_In_z_ const char *file_name, _In_opt_ const ebpf_program_type_t *program_type, _In_opt_ const ebpf_attach_type_t *attach_type, _Outptr_ struct _ebpf_object **object, _Out_ fd_t *program_fd, _Outptr_result_maybenull_z_ const char **log_buffer)
/home/runner/work/ebpf-for-windows/ebpf-for-windows/include/ebpf_api.h:391: warning: The following parameter of ebpf_program_load(_In_z_ const char *file_name, _In_opt_ const ebpf_program_type_t *program_type, _In_opt_ const ebpf_attach_type_t *attach_type, _Outptr_ struct _ebpf_object **object, _Out_ fd_t *program_fd, _Outptr_result_maybenull_z_ const char **log_buffer) is not documented:
  parameter 'object'
```

--- a/include/ebpf_api.h
+++ b/include/ebpf_api.h
@@ -114,6 +114,8 @@ extern "C"
      * @param[in] key_size Key size.
      * @param[in] value_size Value size.
      * @param[in] max_entries Maximum number of entries in the map.
+     * @param[in] map_flags Map flags.
+     * @param[out] handle Pointer to map handle.
      *
      * @retval EBPF_SUCCESS Map created successfully.
      * @retval EBPF_ERROR_NOT_SUPPORTED Unsupported map type.
@@ -408,7 +410,7 @@ extern "C"
      * @param[in] execution_type The execution type to use for this program. If
      *  EBPF_EXECUTION_ANY is specified, execution type will be decided by a
      *  system-wide policy.
-     * @param[out] ebpf_object Returns pointer to ebpf_object object. The caller
+     * @param[out] object Returns pointer to ebpf_object object. The caller
         is expected to call ebpf_object_close() at the end.
      * @param[out] program_fd Returns a file descriptor for the first program.
      *  The caller should not call _close() on the fd, but should instead use
@@ -485,7 +487,7 @@ extern "C"
     /**
      * @brief Fetch fd for a map object.
      *
-     * @param[in] program Pointer to eBPF map.
+     * @param[in] map Pointer to eBPF map.
      * @return fd for the map on success, ebpf_fd_invalid on failure.
      */
     fd_t

--- a/include/ebpf_helpers.h
+++ b/include/ebpf_helpers.h
@@ -31,6 +31,7 @@ EBPF_HELPER(void*, ebpf_map_lookup_element, (ebpf_map_definition_t * map, void* 
  * @param[in] map Map to update.
  * @param[in] key Key to use when searching and updating the map.
  * @param[in] value Value to insert into the map.
+ * @param[in] flags Map flags.
  * @retval EBPF_SUCCESS The operation was successful.
  * @retval EBPF_NO_MEMORY Unable to allocate resources for this
  *  entry.

